### PR TITLE
update to gradle 7

### PR DESCRIPTION
**Pull request by cdietrich**
update to gradle 7
https://github.com/eclipse/xtext/issues/1548

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Review by oehme — APPROVED** [review]

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (szarnekow) [inline .classpath:1]: Why this?

**Review by szarnekow — COMMENTED** [review]
Code changes look good at a first glance. I'll give it a try locally, too.

File: .classpath (hunk 1 above)
Comment (cdietrich) [reply .classpath:1]: no idea buildship doing buildship things

Comment (szarnekow) [reply .classpath:1]: Sadness. More files that will be touched after a refresh gradle projects. -
Should we push these?

Comment (cdietrich) [reply .classpath:1]: i have pushed all if seen correct

**Pull request merged**